### PR TITLE
Inline as multi-line object literals in arguments

### LINKED ISSUE #54: Single object literal in argument should prefer to be inline multi-line when exceeding line width
**Input Code**

```ts
asdfasdf({ name: "Test", aliases: "testtttttttttttttttttttttttttttttttttttttttt" });
```

**Expected Output**

```ts
asdfasdf({
    name: "Test",
    aliases: "testttttttttttttttttttttttttttttttttttttttttttt",
});
```

**Actual Output**

```ts
asdfasdf(
    {
        name: "Test",
        aliases: "testttttttttttttttttttttttttttttttttttttttttttt",
    }
);
```

### PR DESCRIPTION
**Pull request by kdy1**
Closes #54.

### CODE REVIEW

**Comment by kdy1** [comment]
> _No description provided._

File: src/parsing/parser.rs [hunk 1]
```diff
@@ -5129,6 +5140,7 @@ fn parse_parameters_or_arguments<'a, F>(opts: ParseParametersOrArgumentsOptions<
                 multi_line_options: parser_helpers::MultiLineOptions::surround_newlines_indented(),
                 force_possible_newline_at_start: is_parameters,
                 node_sorter: None,
+                prefer_inline_multi_line: !is_parameters,
```
Comment (dsherret) [inline src/parsing/parser.rs:5143]: Everything looks good! Thanks a lot, @kdy1! I am only not sure about this. I'll have to think about it a little bit and will merge over the next few days. I think there are probably some scenarios this affects that aren't properly tested, but I will add some tests if I think of anything :)

File: src/parsing/parser.rs [hunk 2]
```diff
@@ -5322,6 +5334,7 @@ struct ParseSeparatedValuesOptions<'a> {
     multi_line_options: parser_helpers::MultiLineOptions,
     force_possible_newline_at_start: bool,
     node_sorter: Option<Box<dyn Fn((usize, Option<&Node<'a>>), (usize, Option<&Node<'a>>), &Module<'a>) -> std::cmp::Ordering>>,
+    prefer_inline_multi_line: bool,
```
Comment (dsherret) [inline src/parsing/parser.rs:5337]: I think this should maybe be `use_new_line_groups` or something related.

**Review by dsherret — APPROVED** [review]

File: src/parsing/parser.rs (hunk 2 above)
Comment (kdy1) [reply src/parsing/parser.rs:5337]: Got it. It seems much intuitive. I'll change it soon.

File: src/parsing/parser.rs (hunk 1 above)
Comment (kdy1) [reply src/parsing/parser.rs:5143]: Thanks! I'll also try to find such cases.
I think some call expressions with complex arguments can be candidate.

**Comment by dsherret** [comment]
Thanks again, @kdy1! I've merged this now.

**Comment by dsherret** [comment]
Just a heads up that I inverted this to be opt-in instead of opt-out https://github.com/dprint/dprint-plugin-typescript/commit/c8f1e4fdf2b153654808021c6f477d98205aa1b4

**Comment by kdy1** [comment]
Great, thanks!

**Pull request merged**